psa-crypto/src/types/key.rs

```diff
@@ -255,6 +257,53 @@ impl Attributes {
     pub(crate) fn reset(attributes: &mut psa_crypto_sys::psa_key_attributes_t) {
         unsafe { psa_crypto_sys::psa_reset_key_attributes(attributes) };
     }
+
+    /// Gets the attributes for a given key ID
+    ///
+    /// The `Id` structure can be created with the `from_persistent_key_id` constructor on `Id`.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use psa_crypto::operations::key_management;
+    /// # use psa_crypto::types::key::{Attributes, Type, Lifetime, Policy, UsageFlags};
+    /// # use psa_crypto::types::algorithm::{AsymmetricSignature, Hash};
+    /// # let mut attributes = Attributes {
+    /// #     key_type: Type::RsaKeyPair,
+    /// #     bits: 1024,
+    /// #     lifetime: Lifetime::Volatile,
+    /// #     policy: Policy {
+    /// #         usage_flags: UsageFlags {
+    /// #             sign_hash: true,
+    /// #             sign_message: true,
+    /// #             verify_hash: true,
+    /// #             verify_message: true,
+    /// #             ..Default::default()
+    /// #         },
+    /// #         permitted_algorithms: AsymmetricSignature::RsaPkcs1v15Sign {
+    /// #             hash_alg: Hash::Sha256.into(),
+    /// #         }.into(),
+    /// #     },
+    /// # };
+    /// psa_crypto::init().unwrap();
+    /// let my_key_id = key_management::generate(attributes, None).unwrap();
+    /// //...
+    /// let key_attributes = Attributes::from_key_id(my_key_id);
+    /// ```
+    pub fn from_key_id(key_id: Id) -> Result<Self> {
```

Comment:
Would it be worth having a test for this?

Reply:
As there is an example, let's say it is fine for now but think about adding one in the future 🤡